application/src/main/java/org/ehrbase/application/config/SecurityConfig.java

```diff
@@ -64,32 +64,44 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
 
         // For Basic Auth: assigns specific roles to specific users. Enables conditional handling in configure()
         auth.inMemoryAuthentication()
-                    .withUser(securityYAMLConfig.getAuthUser())
-                    .password(passwordEncoder().encode(securityYAMLConfig.getAuthPassword()))
-                    .roles(USER)
+                .withUser(securityYAMLConfig.getAuthUser())
+                .password(passwordEncoder().encode(securityYAMLConfig.getAuthPassword()))
+                .roles(USER)
                 //.authorities("ROLE_USER");
                 .and()
-                    .withUser(securityYAMLConfig.getAuthAdminUser())
-                    .password(passwordEncoder().encode(securityYAMLConfig.getAuthAdminPassword()))
-                    .roles(ADMIN);
+                .withUser(securityYAMLConfig.getAuthAdminUser())
+                .password(passwordEncoder().encode(securityYAMLConfig.getAuthAdminPassword()))
+                .roles(ADMIN);
     }
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         switch (securityYAMLConfig.getAuthType()) {
             case BASIC:
+                logger.info(LOG_SEPARATOR);
                 logger.info("Using basic authentication.");
-                logger.info(formatter.format(
-                        "Username: %s Password: %s", securityYAMLConfig.getAuthUser(), securityYAMLConfig.getAuthPassword()
-                ).toString());
+                logger.info(String.format(
+                        "User-Username: %s", securityYAMLConfig.getAuthUser()
+                ));
+                logger.info(String.format(
+                        "User-Password: %s", securityYAMLConfig.getAuthPassword()
+                ));
+                logger.info(LOG_SEPARATOR);
+                logger.info(String.format(
+                        "Admin-Username: %s", securityYAMLConfig.getAuthAdminUser()
+                ));
+                logger.info(String.format(
+                        "Admin-Password: %s", securityYAMLConfig.getAuthAdminPassword()
+                ));
+                logger.info(LOG_SEPARATOR);
```

Comment:
username and password should never! be written in the log.

Please remove

Reply:
Removed mentioned logging.

Also disabled actuator endpoints by default and added instruction on docker documentation how to enabled endpoints.